src/interrogate/coverage.py

```diff
@@ -128,11 +128,7 @@ def _filter_files(self, files):
                 basename = os.path.basename(f)
                 if basename == "__init__.py":
                     continue
-            maybe_excluded_dir = any(
-                [f.startswith(exc) for exc in self.excluded]
-            )
-            maybe_excluded_file = f in self.excluded
-            if any([maybe_excluded_dir, maybe_excluded_file]):
+            if any([f.startswith(os.path.normpath(exc)) for exc in self.excluded]):
```

Comment:
hey @oriash93 - thanks for the PR! Glad it was just a small change :) It looks like formatting is failing the build. Can you run `tox -e lint` locally to take care of that, then push your changes up?